eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/searchlist/SearchModulesFactory.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.searchlist;
+
+import android.app.Activity;
+
+import com.schedjoules.eventdiscovery.framework.list.changes.notifying.ChangeableListItems;
+import com.schedjoules.eventdiscovery.framework.list.sectioned.SectionableListItem;
+import com.schedjoules.eventdiscovery.framework.list.sectioned.SectionedChangeableListProxy;
+import com.schedjoules.eventdiscovery.framework.utils.Factory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Factory for creating {@link SearchModule}s for a search list.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class SearchModulesFactory implements Factory<List<SearchModule>>
```

Comment:
The example for a search list with elements that react differently for selection would be the Google search on the phone, it can show both apps and web search results as well.
But you are right, it's probably not worth keeping, maintaining unnecessarily, we can create if ever we need. So I removed it and removed `Selectable..` prefix from that one, so this is the default usage now.